### PR TITLE
fix navbar mobile menu toggle and styling

### LINKED ISSUE #161: Navbar burger doesn't toggle menu (mobile)
Navbar's burger button doesn't toggle the menu (via class "is-active").

todo:
- toggle menu via burger
- add property to show/hide menu from outside?
- add menu show/hide event?

### PR DESCRIPTION
**Pull request by christiansiegel**
Fixes #161 

(includes unmerged changes of PR #159 and #160)

There are still two open issues:
1. `bal-track-line` breaks the mobile menu + burger location (see header in light mode)
2. menu item colors are not styled correctly (see navbar.scss)

### CODE REVIEW

**Review by hirsch88 — APPROVED** [review]

File: src/components/navbar/navbar.tsx [hunk 1]
```diff
@@ -1,53 +1,62 @@
...
-             role="navigation" aria-label="main navigation">
+        <nav class={'navbar is-spaced' + (this.light ? ' is-white' : ' is-info')}
+             role="navigation"
+             aria-label="main navigation">
+          <div class="bal-track-line"
+               style={{
+                 position: 'absolute',
+                 top: '0',
+                 left: '0',
+                 display: !this.light ? 'none' : '',
```
Comment (christiansiegel) [inline src/components/navbar/navbar.tsx:40]: why even render the `div` if `!this.light`?

Comment (hirsch88) [reply src/components/navbar/navbar.tsx:40]: this is the baloise track line we need that for the light style

Comment (hirsch88) [reply src/components/navbar/navbar.tsx:40]: in the normal mode we hide it

**Pull request merged**